lib/llvm/import-llvm

```diff
@@ -42,13 +45,18 @@ Support_imports = [
 Support_imports += [
     'COFF', 'ConvertUTF', 'ConvertUTFWrapper', 'Debug', 'FileSystem',
     'FileUtilities', 'Host', 'Locale', 'Path', 'Process', 'Program', 'SMLoc',
-    'SourceMgr', 'Signals', 'Unicode', 'UnicodeCharRanges']
+    'SourceMgr', 'Signals', 'Unicode', 'UnicodeCharRanges', 'CommandLine',
+    'MachO', 'Error', 'StringSaver', 'thread', 'TargetParser',
+    'ARMBuildAttributes', 'ARMTargetParser', 'AArch64TargetParser']
```

Comment:
I *really* would prefer not to bring in target specific parser things. These are not well things that make sense in our version of general purpose support library.

Unfortunately, resolving this may require making some knife edits in the code to be able to detangle `Triple`. Do you know what it was that forced this to be brought in?